assessments/27-28/week-03/solutions.md

```diff
@@ -0,0 +1,92 @@
+# <img src="https://cloud.githubusercontent.com/assets/7833470/10899314/63829980-8188-11e5-8cdd-4ded5bcb6e36.png" height="60"> Week 3 Assessment [SOLUTIONS]
+
+1. A user fills out a form and hits "submit". List at least 3 things that might happen next.
+
+  * The page refreshes
+  * The form sends parameters to the server
+  * jQuery prevents the default behavior, and the page DOES NOT refresh
+  * jQuery gets the values from the form input fields
+  * An AJAX request sends the form data to the server asynchronously
+
+2. Give two examples of HTTP status codes and what they mean.
```

Comment:
We did discuss it.  But I doubt many of them got it.  I very carefully and
in two separate lessons have discussed this but think it's still foggy.

On Thu, Mar 31, 2016, 09:16 Brianna Veenstra <notifications@github.com>
wrote:

> In assessments/27-28/week-03/solutions.md
> <https://github.com/SF-WDI-LABS/shared_modules/pull/19#discussion_r58082637>
> :
>
> > @@ -0,0 +1,92 @@
> > +# <img src="https://cloud.githubusercontent.com/assets/7833470/10899314/63829980-8188-11e5-8cdd-4ded5bcb6e36.png" height="60"> Week 3 Assessment [SOLUTIONS]
> > +
> > +1. A user fills out a form and hits "submit". List at least 3 things that might happen next.
> > +
> > +  * The page refreshes
> > +  * The form sends parameters to the server
> > +  * jQuery prevents the default behavior, and the page DOES NOT refresh
> > +  * jQuery gets the values from the form input fields
> > +  * An AJAX request sends the form data to the server asynchronously
> > +
> > +2. Give two examples of HTTP status codes and what they mean.
>
> Don't know if 27 went over this, perhaps @tgaff <https://github.com/tgaff>
> did in HTTP lesson?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/SF-WDI-LABS/shared_modules/pull/19/files/4a52ffbd77e6168ea7c4f6f20c4c32592b7da3da#r58082637>
>